Assets/Engine/Interactions/IntentManager.cs

```diff
@@ -19,13 +19,27 @@ public class IntentManager : MonoBehaviour
     public Color colorHarm;
     public Color colorHelp;
 
+    private Button IntentButton;
```

Comment:
All non-static fields are written camelCase.

Reply:
Should be solved now !